app.py

```diff
@@ -1,11 +1,14 @@
-from flask import Flask, redirect, render_template
+import re
+from flask import Flask, redirect, render_template, request
 
 from src.repositories.movie_repository import get_movie_repository
 
 app = Flask(__name__)
 
 movie_repository = get_movie_repository()
 
+form_data = {}
```

Comment:
I am confused on what this is used for.

Reply:
Dictionary was for the commented code. If movie_repository.create_movie works for you guys to access form data, then I will delete that and the commented code.